### PR TITLE
Adds instructions to change the time using systemd-timesyncd.service.

### PR DESCRIPTION
**Pull request by iphutch**
Signed-off-by: ihutchin <ianeta.hutchinson@intel.com>

### CODE REVIEW

File: source/clear-linux/guides/maintenance/time.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+Set the time
+############
+
+In |CL|, it is not possible to set the time manually, via `timedatectl`
+nor `RTC` mode. |CLOSIA| instead uses the `systemd-timesyncd.service` daemon
+to synchronize time.
+
+This section provides instructions to reset the time in your |CL| system when
+the default network time protocol (NTP) servers cannot be reached. The default
```
Comment (rcaballeromx) [inline source/clear-linux/guides/maintenance/time.rst]: Use the :abbr: role

File: source/clear-linux/guides/maintenance/time.rst [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+NTP servers are configured as
+time1.google.com, time2.google.com, time3.google.com, and time4.google.com.
+
+#. Install the `sysadmin-basic` bundle:
+
+   .. code-block:: console
+
+      swupd bundle-add sysadmin-basic
+
+#. Set your timezone (this example uses Los Angeles).
```
Comment (rcaballeromx) [inline source/clear-linux/guides/maintenance/time.rst]: Avoid parenthesis.

File: source/clear-linux/guides/maintenance/time.rst [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+   .. code-block:: console
+
+      timedatectl set-timezone America/Los_Angeles
+
+   .. note::
+      To see a list of timezones, enter:
+      `timedatectl list-timezones | grep <locale>`
+
+#. Create a `/etc/systemd/` directory.
```
Comment (rcaballeromx) [inline source/clear-linux/guides/maintenance/time.rst]: Use the :file: role for directories.

File: source/clear-linux/guides/maintenance/time.rst [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+   .. code-block:: console
+
+      timedatectl set-ntp true
+
+.. note::
+
+   Use the :command:`timedatectl status` command to check the service status.
+   To restart the `timesyncd` daemon, enter :command:`systemctl restart systemd-timesyncd`
+   into your terminal emulator.
```
Comment (rcaballeromx) [inline source/clear-linux/guides/maintenance/time.rst]: Add a closing statement along the lines: "Congratulations! You have successfully set up the time in you |CL| system."

**Comment by phmccarty** [comment]
The instructions look correct to me. +1

**Pull request merged**